navigation/navigation.css

```diff
@@ -0,0 +1,206 @@
+.navigation{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 5rem;
+    box-sizing: border-box;
+    background-color: var(--nav-color);
+    /* position: fixed; */
+    width: 100%;
+    padding: 1rem;
```

Comment:
I just checked, this class is being used in the documentation navbar and example navbar so the height of 5rem is causing an issue, you can remove that